src/schema/v2/me/myBids.ts

```diff
@@ -122,15 +122,15 @@ export const MyBids: GraphQLFieldConfig<void, ResolverContext> = {
     ])
 
     // Map over response to gather all sale IDs
-    const causalityLots = causalityResponse.lotStandingConnection.edges.map(
-      ({ node }) => node
-    )
+    const causalityLots = (
+      causalityResponse?.lotStandingConnection?.edges ?? []
+    ).map(({ node }) => node)
```

Comment:
This also fixes that weird issue you were running into @dzucconi that i was also hitting around some kind of staging stitching problem